topics/07_web_ui_testing/example_of_test_suites_and_po/README.md

```diff
@@ -0,0 +1,47 @@
+# Introduction
+This folder contains files with PO and test suites to demo different implementation of Web UI automation tests
+
+
+## Files
+
+1)
+selenium_test_demo.py:
```

Comment:
Put the file names in the same line as the number items and add a backtick to better style. Example:
```
1. `selenium_test_demo.py`: It is a simple example of selenium webdriver test. CLI command:
   \```bash
   $ python .\selenium_test_demo.py
   \```
```

The above slashes were only to demonstrate. Do this change for all of the next ones.

Reply:
Done.
Review it, please.